Lib.AspNetCore.ServerTiming/Processors/DevelopmentEnvironmentBasedProcessor.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Lib.AspNetCore.ServerTiming.Http.Headers;
+using Microsoft.AspNetCore.Http;
+using System.Collections.Generic;
+
+namespace Lib.AspNetCore.ServerTiming.Processors
+{
+    /// <summary>
+    /// Base class for processors whose behaviour changes in the development environment
+    /// </summary>
+    public abstract class DevelopmentEnvironmentBasedProcessor : IServerTimingProcessor
+    {
+        /// <summary>
+        /// True if running in the development environment
+        /// </summary>
+        protected readonly bool IsDevelopment = false;
```

Comment:
What do you think about exposing `IsStaging` and `IsProduction` on this base class? Currently if someone wants to utilize one of those two it creates a necessity to implement from scratch.